### PR TITLE
Tests all the velero versions

### PR DESCRIPTION
**Pull request by claudiubelu**
Adds integration tests for all the currently supported velero versions.

### CODE REVIEW

File: tests/integration/test_velero.py [hunk 1]
```diff
@@ -136,4 +146,16 @@ def test_integration_velero(module_instance: harness.Instance):
         backup_name,
     )
 
-    k8s_util.wait_for_deployment(module_instance, "nginx-deployment", "nginx-example")
+    k8s_util.wait_for_deployment(instance, "nginx-deployment", "nginx-example")
+
+
+def test_integration_velero_1_13_2(function_instance: harness.Instance):
```
Comment (bschimke95) [inline tests/integration/test_velero.py]: Let's make this a parameterized test to align with the other Rocks, wdyt?

**Review by bschimke95 — COMMENTED** [review]
looks good, only minor comment

File: tests/integration/test_velero.py (hunk 1 above)
Comment (claudiubelu) [reply tests/integration/test_velero.py]: Done.

**Review by bschimke95 — APPROVED** [review]
LGTM

**Pull request merged**